app/jobs/calendar_importer/parsers/resident_advisor.rb

```diff
@@ -0,0 +1,104 @@
+# frozen_string_literal: true
+
+module CalendarImporter::Parsers
+  class ResidentAdvisor < Base
+    NAME = 'Resident Advisor'
+    KEY = 'residentadvisor'
+    DOMAINS = %w[ra.co].freeze
+    RA_ENDPOINT = 'https://ra.co/graphql'
+    RA_REGEX = %r{^https://ra\.co/(promoters|clubs)/(\d+)$} # Accept club or promoter URLs
```

Comment:
do we want this in a var somewhere or are these obviously the only 2 options we need to handle?

Reply:
I meant do we want to be able to set the value in config for people who use RA for something other than promoters or clubs or if they only use one of them for example.

Reply:
i still dont get you, apols

Reply:
No problem. It sounds like hard code is the way to go because it is the same for everyone all the time.